### ISSUE TITLE
Void pixels in ground truth labels

### ISSUE THREAD

**Issue by ShanyCohen** [TUI-NICR/ESANet#31]
Hello,

I'm trying to figure out how the void pixels affect the mIou and the loss computation.
The labels are being decremented by 1 in the code so that class numbers in the prediction and the ground truth labels will match. 
So to my understanding, there are pixels with the value -1 in the label image but the prediction doesn't include void pixels.
Are those pixels being ignored or being considered as true/false predictions?

Thanks,
Shany

**Comment by mona0809** [comment]
Void is ignored during both training and mIoU.
For cross entropy loss the index -1 is ignored (see: https://github.com/TUI-NICR/ESANet/blob/main/src/utils.py#L27-L30)
For mIoU, both prediction and ground truth are filtered, such that only those pixels are taken into account, that are not void in ground truth. (see: https://github.com/TUI-NICR/ESANet/blob/main/train.py#L437-L460)
